index-map-area.html

```diff
@@ -31,19 +31,19 @@
         /* border: none; */
       }
       
-      /* Pre-style to avoid FOUC of inline layer- and fallback content. */
-      map[is="web-map"]:not(:defined) + img[usemap],
-      map[is="web-map"]:not(:defined) > :not(area):not(.mapml-web-map) {
-        display: none;
-      }
-      
       /* Pre-style to avoid Layout Shift. */
       map[is="web-map"]:not(:defined) {
         display: inline-block;
         contain: size;
         contain-intrinsic-size: 304px 154px;
       }
       
+      /* Pre-style to avoid FOUC of inline layer- and fallback content. */
+      map[is="web-map"]:not(:defined) + img[usemap],
+      map[is="web-map"]:not(:defined) > :not(area):not(.mapml-web-map) {
+        display: none;
+      }
+      
       /* Ensure inline layer content is hidden if custom/built-in elements isn't
       supported, or if javascript is disabled. This needs to be defined separately
       from the above, because the `:not(:defined)` selector invalidates the entire
```

Comment:
The comment: "This needs to be defined separately from the above" now correctly refers to the other `display: none` declaration, which is now directly above. The order of styles should have been considered in https://github.com/Maps4HTML/Web-Map-Custom-Element/pull/487.